Nickvision.Aura/Aura.cs

```diff
@@ -44,15 +40,13 @@ private Aura(string id, string name, string shortName, string description)
     /// </summary>
     /// <param name="id">ID for AppInfo</param>
     /// <param name="name">Name for AppInfo</param>
-    /// <param name="shortName">ShortName for AppInfo</param>
-    /// <param name="description">Description for AppInfo</param>
-    public static void Init(string id, string name, string shortName, string description)
+    public static void Init(string id, string name)
```

Comment:
Maybe make `shortName` and `description` option in the method that way we don't need to add two more lines:
```csharp
public static void Init(string id, string name, string shortName = "", string description = "")
{
   ...
   _instance = new Aura(id, name) 
   {
       ShortName = shortName,
       Description = description
   };
   ...
}
```

Reply:
what's the point of this, except of backward compatibility? Short name and description are as optional as version and other fields, there's nothing special in them and so no reason to keep that legacy.

Reply:
Yeah that's true... forget it then